components/meter/meter-radial.js

```diff
@@ -0,0 +1,56 @@
+import '../colors/colors.js';
+import { css, html, LitElement } from 'lit-element/lit-element.js';
+import { heading4Styles } from '../typography/styles.js';
+
+class MeterRadial extends LitElement {
+	static get properties() {
+		return {
+			value: { type: Number },
+			max: { type: Number }
+		};
+	}
+
+	static get styles() {
+		return [ heading4Styles, css`
+		:host {
+			display: inline-block;
+			width: 86px;
```

Comment:
Only question here is whether the width is dependent on the base font size? Because the user can adjust this between different sizes. A `rem`-based size might be better if that's a problem. I'd also test out with browser zoom.

Reply:
I tested with the browser zoom. The `rem` and `px` seemed to have the same effect. But you are correct I should have used the `rem` value here.